Multisig/UI/Assets/AssetsViewModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AssetsViewModel.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 13.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+struct TokenBalance: Identifiable {
+    var id: String {
+        return address
+    }
+    var imageURL: URL {
+        return URL(string: "https://gnosis-safe-token-logos.s3.amazonaws.com/\(address).png")!
+    }
+    let address: String
+    let balance: String
+    let balanceUsd: String
+}
+
+class AssetsViewModel: ObservableObject {
+    @Published var balances = [TokenBalance]()
+    @Published var isLoading: Bool = true
+    @Published var errorMessage: String? = nil
+
+    private var subscribers = Set<AnyCancellable>()
+
+    init() {
+        let request = Safe.fetchRequest().selected()
+        let selectedSafe = try! CoreDataStack.shared.viewContext.fetch(request).first!
+        isLoading = true
+        Just(selectedSafe.address!)
+            .setFailureType(to: Error.self)
```

Comment:
good find! Can you find other places in code where I used `tryMap` or something - to also use this failure type instead?

Reply:
I created a debt issue: https://app.zenhub.com/workspaces/safe-multisig---mobile-5c1b98250e13551e8aae81eb/issues/gnosis/safe-ios/132
Sorry, now I want to merge this PR asap.